README.md

```diff
@@ -1,11 +1,35 @@
-GenAI contains pipelines that implement image and text generation tasks. The implementation exploits OpenVINO capabilities to optimize the pipelines. Each sample covers a family of models and suggests that its implementation can be modified to adapt for a specific need.
+<!---
+- [Repository Description](#repository-description))
+   - [Pipeline Samples](#pipeline-samples)
+   - [License](#license)
+- [Requirements](#requirements)
+-->
+## GenAI Pipeline Repository
 
-> Note  
+The GenAI repository contains pipelines that implement image and text generation tasks.
+The implementation uses OpenVINO capabilities to optimize the pipelines. Each sample covers
+a family of models and suggests certain modifications to adapt the code to specific needs.
+
+> Note
 This project is not for production use.
 
+### Pipeline Samples 
+ - [image generation by stable diffusion](image_generation/stable_diffusion_1_5/cpp)
+ - [large language model demo](llm/cpp)
+ - [Benchmarking script for large language models](llm_bench/python)
+
+### License
+
+The GenAI repository is licensed under [Apache License Version 2.0](LICENSE).
+By contributing to the project, you agree to the license and copyright terms therein and release
+your contribution under these terms.
+
+## Requirements
+
+Requirements may vary for different samples. See respective readme files for more details.
 Every pipeline requires https://github.com/openvinotoolkit/openvino for C++ to be installed.
```

Comment:
For the python sample, C++ OpenVINO is not required.

Reply:
Yeah, I was wondering if this statement was still true :) Thanks.